app/src/main/res/layout/dashboard_referrer_breakdown_layout.xml

```diff
@@ -0,0 +1,205 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
+  android:orientation="vertical"
+  android:layout_width="match_parent"
+  android:layout_height="wrap_content"
+  android:id="@+id/referrer_breakdown_section_layout">
+
+  <TextView
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:text="@string/dashboard_graphs_referrers_title_referrers"
+    style="@style/Headline"
+    android:paddingStart="@dimen/grid_3" />
+
+  <LinearLayout
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:orientation="horizontal"
+    android:id="@+id/referrer_breakdown_chart_layout">
+
+    <com.kickstarter.ui.views.ReferrerBreakdownView
+      android:id="@+id/referrer_breakdown_view"
+      android:layout_margin="@dimen/grid_2"
+      android:layout_width="@dimen/grid_21"
+      android:layout_height="@dimen/grid_21" />
+
+    <LinearLayout
+      android:layout_width="wrap_content"
+      android:layout_height="match_parent"
+      android:layout_margin="@dimen/grid_2"
+      android:orientation="vertical">
+
+      <LinearLayout
+        android:id="@+id/pledged_via_kickstarter_layout"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal"
+        android:layout_weight="1">
+
+        <ImageView
+          android:id="@+id/percent_via_kickstarter_circle_text_view"
+          android:contentDescription="@string/dashboard_graphs_referrers_pledged_via_kickstarter"
```

Comment:
@luoser wanted to make sure this was finesince I haven't really dealt with accessibility on android before.  These are just the bullets next to the graph but since they are image views, the linter wants them to have a description.  I'm just reusing the strings that we use in the textviews below for the description.

Reply:
ah yes, the move here would actually to ignore the `contentDescription` since the circle imageview is just a visual indicator that doesn't give the user any new / meaningful information that the textviews provide

```xml
tools:ignore="ContentDescription"
```

should appease the linter!

Reply:
ah amazing ty